wgpu/src/backend/webgpu.rs

```diff
@@ -1562,6 +1562,38 @@ impl dispatch::InstanceInterface for ContextWebGpu {
         // Devices are automatically polled.
         true
     }
+
+    #[cfg(feature = "wgsl")]
+    fn wgsl_language_features(&self) -> std::collections::HashSet<crate::WGSLLanguageFeatures> {
+        if let Some(gpu) = &self.gpu {
```

Comment:
Nit: convert this to an early out

Reply:
Now that I changed to bitflags this is no longer relevant.